meta/saisanitycheck.c

```diff
@@ -2286,6 +2290,21 @@ void check_attr_acl_capability(
     }
 }
 
+void check_attr_acl_chain_list(
```

Comment:
you are making sure that acl chain list here is always read only, i think this check is not necessary

Reply:
https://github.com/opencomputeproject/SAI/pull/1815/commits/f56ef7b31276c8228817804428421cbf2870c1db